source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to sync data between a replica set and a 
+sharded cluster using :program:`mongosync`. To perform a sync between two
+replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync, define your source replica set, your 
+destination sharded cluster, and an administrative user that has permissions in 
+both clusters. You must also download ``mongosync``. To learn more, see 
+the :ref:`setup <c2c-quickstart-setup>` instructions.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Connect the replica set and sharded cluster. 
+
+      ``mongosync`` must create an initial connection to the source and 
+      destination clusters before you can start to sync data.
+      Follow the :ref:`connection instructions <c2c-conn-top-level>` for your 
+      cluster architecture to connect to the :binary:`mongos` instance in your 
+      cluster.
+
+      For an example of the ``mongosync`` connection command, see 
+      :ref:`Initialize mongosync <c2c-initialize>`.
```

Comment:
[question] since the linked example of the mongosync connection command is fairly short, what do you think about including it here instead of requiring the user to click the link?

Reply:
Alternatively, you could rephrase the sentence "For more information, see Initialize mongosync" to refer to more than just the example command.

Reply:
I am worried about this example because it is for a replica set to a replica set. In my mongosync behavior ticket, I created a more general example format for the command. I hope that gets merged first, and then I will link that example instead. I will leave this comment unresolved so that I can come back to it.

Reply:
Put in more generic code example so users will not have to click the link.